lib/experimental-default-theme.json

```diff
@@ -4,102 +4,126 @@
 					},
 					{
-						"slug": "electric-grass",
-						"gradient": "linear-gradient(135deg,rgb(202,248,128) 0%,rgb(113,206,126) 100%)"
+						"name": "Luminous vivid orange to vivid red",
+						"gradient": "linear-gradient(135deg,rgba(255,105,0,1) 0%,rgb(207,46,46) 100%)",
+						"slug": "luminous-vivid-orange-to-vivid-red"
 					},
 					{
-						"slug": "light-green-cyan-to-vivid-green-cyan",
-						"gradient": "linear-gradient(135deg,rgb(122,220,180) 0%,rgb(0,208,130) 100%)"
+						"name": "Very light gray to cyan bluish gray",
+						"gradient": "linear-gradient(135deg,rgb(238,238,238) 0%,rgb(169,184,195) 100%)",
+						"slug": "very-light-gray-to-cyan-bluish-gray"
 					},
 					{
-						"slug": "luminous-dusk",
-						"gradient": "linear-gradient(135deg,rgb(255,203,112) 0%,rgb(199,81,192) 50%,rgb(65,88,208) 100%)"
+						"name": "Cool to warm spectrum",
+						"gradient": "linear-gradient(135deg,rgb(74,234,220) 0%,rgb(151,120,209) 20%,rgb(207,42,186) 40%,rgb(238,44,130) 60%,rgb(251,105,98) 80%,rgb(254,248,76) 100%)",
+						"slug": "cool-to-warm-spectrum"
 					},
 					{
-						"slug": "luminous-vivid-amber-to-luminous-vivid-orange",
-						"gradient": "linear-gradient(135deg,rgba(252,185,0,1) 0%,rgba(255,105,0,1) 100%)"
+						"name": "Blush light purple",
+						"gradient": "linear-gradient(135deg,rgb(255,206,236) 0%,rgb(152,150,240) 100%)",
+						"slug": "blush-light-purple"
 					},
 					{
-						"slug": "luminous-vivid-orange-to-vivid-red",
-						"gradient": "linear-gradient(135deg,rgba(255,105,0,1) 0%,rgb(207,46,46) 100%)"
+						"name": "Blush bordeaux",
+						"gradient": "linear-gradient(135deg,rgb(254,205,165) 0%,rgb(254,45,45) 50%,rgb(107,0,62) 100%)",
+						"slug": "blush-bordeaux"
 					},
 					{
-						"slug": "midnight",
-						"gradient": "linear-gradient(135deg,rgb(2,3,129) 0%,rgb(40,116,252) 100%)"
+						"name": "Luminous dusk",
+						"gradient": "linear-gradient(135deg,rgb(255,203,112) 0%,rgb(199,81,192) 50%,rgb(65,88,208) 100%)",
+						"slug": "luminous-dusk"
 					},
 					{
-						"slug": "pale-ocean",
-						"gradient": "linear-gradient(135deg,rgb(255,245,203) 0%,rgb(182,227,212) 50%,rgb(51,167,181) 100%)"
+						"name": "Pale ocean",
+						"gradient": "linear-gradient(135deg,rgb(255,245,203) 0%,rgb(182,227,212) 50%,rgb(51,167,181) 100%)",
+						"slug": "pale-ocean"
 					},
 					{
-						"slug": "very-light-gray-to-cyan-bluish-gray",
-						"gradient": "linear-gradient(135deg,rgb(238,238,238) 0%,rgb(169,184,195) 100%)"
+						"name": "Electric grass",
+						"gradient": "linear-gradient(135deg,rgb(202,248,128) 0%,rgb(113,206,126) 100%)",
+						"slug": "electric-grass"
 					},
 					{
-						"slug": "vivid-cyan-blue-to-vivid-purple",
-						"gradient": "linear-gradient(135deg,rgba(6,147,227,1) 0%,rgb(155,81,224) 100%)"
+						"name": "Midnight",
+						"gradient": "linear-gradient(135deg,rgb(2,3,129) 0%,rgb(40,116,252) 100%)",
+						"slug": "midnight"
```

Comment:
How about i18n here? Is the idea to keep the "double registration" for now (theme.json and php)?